File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.length() != t.length()) return false;
        vector<int>v(26, 0);
        for(int i = 0; i< s.length(); i++){
            v[s[i]-'a'] += 1;
        }
        for(int i = 0; i<t.length(); i++){
            v[t[i]-'a'] -= 1;
        }
        for(int i = 0; i< 26; i++){
            if(v[i] != 0) return false;
        }
        return true;
    }
};

File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        lst1 = [0]*26
        lst2 = [0]*26

        for i in s:
            lst1[ord(i) - 97] += 1
        
        for i in t:
            lst2[ord(i) - 97] += 1
        
        return lst1 == lst2        